e2e/helpers/html_helpers.py

```diff
@@ -0,0 +1,5 @@
+from bs4 import BeautifulSoup
+
+
+def make_soup(content):
+    return BeautifulSoup(content, 'html.parser')
```

Comment:
As we don't encapsulate BeautifulSoup when using it in the test, wouldn't it make sense to have to initialization inline, too, rather than extracted to here? Then it'd at least look familiar to developers who know this library already.

Or is `make_soup` a convention amongst BeatifulSoup users?